tests/unit/test_init.py

```diff
@@ -86,3 +90,55 @@ def test_analyze_does_all_but_run(replica, create_relation):
         assert '().analyze' == replica_methods[2][0]
         replica.assert_called_once()
         create_relation.assert_not_called()
+
+
+@patch('snowshu.core.main.ReplicaFactory.create')
+def test_custom_retry_count_cli_input(create):
+    # test if CLI input is passed to replica.create
+    runner = CliRunner()
+    runner.invoke(main.cli, ('create'))
+    create.assert_called_with(name=ANY, barf=ANY, retry_count=1)
```

Comment:
This bit passes locally but fails in CI, needs investigation

Reply:
I tested it locally - it fails
<img width="1525" alt="image" src="https://user-images.githubusercontent.com/98582037/170554846-bc79d8ab-4dc9-4155-bcc2-5f6c730503cb.png">

Could you please run all unit test cases from docker?
Also, please check the version of pytest

Reply:
Ran as it is supposed to be done 
![image](https://user-images.githubusercontent.com/98971583/170559146-a763a404-59e9-4103-a2ff-3efa2fa6e7f4.png)

Reply:
I reason why it fails - you have no replica.yml file in your root folder
By default it should reside in the root folder

Reply:
it looks like you need to supply the replica.yml file

Reply:
Guess it's fixed, recent run succeeded